spec/arrays.dd

```diff
@@ -580,7 +580,21 @@ Returns an array literal with each element of the literal being the $(D .init) p
         $(TROW $(D .ptr), Returns a pointer to the first element of the array.)
         $(TROW $(D .dup), Create a dynamic array of the same size and copy the contents of the array into it. The copy will have any immutability or const stripped. If this conversion is invalid the call will not compile.)
         $(TROW $(D .idup), Create a dynamic array of the same size and copy the contents of the array into it. The copy is typed as being immutable. If this conversion is invalid the call will not compile.)
-        $(TROW $(D .tupleof), $(ARGS Returns an $(DDSUBLINK spec/template, variadic-templates, expression sequence) of all elements of the array, as with $(D .tupleof) for structs and $(DDSUBLINK spec/class, class_properties, classes).))
+        $(TROW $(D .tupleof), $(ARGS Returns an $(DDSUBLINK spec/template, homogeneous_sequences, lvalue sequence) of each element in the array:
+            $(SPEC_RUNNABLE_EXAMPLE_RUN
+            ---
+            void foo(int, int, int) {}
+
+            int[3] ia;
```

Comment:
Maybe give this an initial value so you're not passing a default-initialized array to `foo`.

Reply:
OK, updated.